### PR TITLE
feat(refactor): refactor create_fam_file

### PR DESCRIPTION
**Pull request by henrikstranneheim**
- renamed set_in_sample_info to set_parameter_in_sample_info
- renamed set_parameter_in_sample_info to set_in_sample_info

### This PR fixes:

- 

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [ ] Code review
- [ ] New code is executed and covered by tests
- [ ] Tests pass

### CODE REVIEW

File: lib/MIP/Pedigree.pm [hunk 1]
```diff
@@ -1723,4 +1656,301 @@ sub _parse_trio_members {
...
+        },
+        sbatch => {
+            method   => \&_write_fam_file_to_sbatch,
+            arg_href => {
+                fam_file_path => $fam_file_path,
+                fam_lines_ref => $fam_lines_ref,
+                filehandle    => $filehandle,
+            },
+        },
+    );
```
Comment (jemten) [inline lib/MIP/Pedigree.pm]: Very nice!

File: lib/MIP/Sample_info.pm [hunk 2]
```diff
@@ -1173,33 +1173,26 @@ sub set_most_complete_vcf {
     return;
 }
 
-sub set_parameter_in_sample_info {
+sub set_in_sample_info {
 
-##Function : Sets parameter to sample info
+##Function : Sets key and value to sample info
```
Comment (jemten) [inline lib/MIP/Sample_info.pm]:
```suggestion
##Function : Sets key and value in sample info
```
My grammar isn't the best but shouldn't it be "in"?

Comment (henrikstranneheim) [reply lib/MIP/Sample_info.pm]: 👍

File: lib/MIP/Pedigree.pm (hunk 1 above)
Comment (henrikstranneheim) [reply lib/MIP/Pedigree.pm]: Thx

**Pull request merged**